src/main/resources/lang/english.lang

```diff
@@ -218,3 +218,8 @@ time:
 io exceptions:
 	unknownhostexception: Cannot connect to %s
 	accessdeniedexception: Access denied for %s
+
+# -- Misc --
+misc:
+	namespacedutils:
+		format: A namespaced key can be formatted as 'namespace:id' or 'id'; Can only contain one ':' to separate the namespace and the id, alphanumeric characters, periods, underscores, and dashes.
```

Comment:
```suggestion
		format: A namespaced key can be formatted as 'namespace:id' or 'id'. It can only contain one ':' to separate the namespace and the id. Only alphanumeric characters, periods, underscores, and dashes can be used.
```
I think it reads a bit weirdly in its current state